odrive_node/src/odrive_can_node.cpp

```diff
@@ -49,6 +51,17 @@ ODriveCanNode::ODriveCanNode(const std::string& node_name) : rclcpp::Node(node_n
 }
 
 void ODriveCanNode::deinit() {
+    if(axis_idle_on_shutdown_) {
+        struct can_frame frame;
+        frame.can_id = node_id_ << 5 | CmdId::kSetAxisState;
+        {
+            std::unique_lock<std::mutex> guard(axis_state_mutex_);
```

Comment:
the mutex was there to guard access to the `axis_state_` variable. However since you're using a const here instead, you can remove the lock.